modules/cudl-data-processing/locals.tf

```diff
@@ -1,4 +1,7 @@
 locals {
+
+  transform-lambda-bucket-names = toset([var.source-bucket-name, var.destination-bucket-name])
+
   source_sns_subscriptions = flatten([
```

Comment:
This local variable could be renamed as it can now contain the dest bucket as well as the source bucket